src/Components/Clock.tsx

```diff
@@ -0,0 +1,50 @@
+import { Component } from 'react';
+
+type Props = {
+  name: string;
+};
+type State = {
+  date: Date;
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    date: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      const date: Date = new Date();
+
+      // eslint-disable-next-line
+      console.log(date.toLocaleTimeString());
+
+      this.setState({ date });
+    }, 1000);
+  }
+
+  componentDidUpdate(prevProps: Props) {
+    const { name: prevName } = prevProps;
+    const { name: newName } = this.props;
+
+    if (prevName === newName) {
+      return;
+    }
+    // eslint-disable-next-line
+    console.log(`The Clock was renamed from ${prevName} to ${newName}`);
```

Comment:
```suggestion
    if (prevName !== newName) {
      // eslint-disable-next-line
      console.log(`The Clock was renamed from ${prevName} to ${newName}`);
    }
```

Reply:
pay attention to the comments, invert your condition
```suggestion
    if (prevName !== newName) {
      // eslint-disable-next-line
      console.log(`The Clock was renamed from ${prevName} to ${newName}`);
    }
```